lib/install/management/virtual_app.go

```diff
@@ -79,7 +79,7 @@ func (d *Dispatcher) createVApp(conf *config.VirtualContainerHostConfigSpec, set
 			Name:      "vSphere Integrated Containers",
 			Vendor:    "VMware",
 			VendorUrl: "http://www.vmware.com/",
-			Version:   "0.0.1",
+			Version:   "0.5.5",
```

Comment:
Hmm.. I am building vic-machine after making the change to see if it shows the right version. I have a couple older binaries on my laptop and I just tried running vic-machine with build 4430 and it seems to fetch the right version. (v0.5.0-4430-f5c412b) I thought it was getting from the same version package. I'll see if I get 0.4.0 as well once I get the binary!